### ISSUE TITLE
close() and closeAll() methods

### ISSUE THREAD

**Issue by f3oall** [f3oall/awesome-notifications#15]
Call of `close()` should close toast
Call of `closeAll()` should close all toasts on the screen

Now there're only two ways to close toast: click on it and wait for end of duration.

If you think that you need a way to hide toasts programmatically, please vote for this issue. When this feature request will have enough votes, I will implement it.

This issue was firstly mentioned in #14.

**Comment by f3oall** [comment]
`closeAll` were implemented in v3.1.0 as `closeToasts`. Read [docs](https://f3oall.github.io/awesome-notifications/docs/toasts/general) to learn more.

There is no need for `close` method because all methods which show toasts return DOM element, so you can use native `remove` to hide it.

PS. Both `closeToasts` and native `remove` methods don't include hiding animation. Let me know if you need these methods implemented in way when they run animation of hiding.

**Comment by manchumahara** [comment]
If anyone is looking to close modal/popup manually

```
//init the modal
var my_modal = new AWN().modal(
    '<b>Custom modal window message</b><br>Class name: `awn-popup-modal-tiny`',
    'modal-tiny'
  );
  
//close/delete the modal
  my_modal.delete();
```
